crate_universe/private/common_utils.bzl

```diff
@@ -121,17 +121,6 @@ def get_rust_tools(repository_ctx, host_triple):
     Returns:
         struct: A struct containing the expected rust tools
     """
-
-    # This is a bit hidden but to ensure Cargo behaves consistently based
-    # on the user provided config file, the config file is installed into
-    # the `CARGO_HOME` path. This is done so here since fetching tools
-    # is expected to always occur before any subcommands are run.
-    if repository_ctx.attr.isolated and repository_ctx.attr.cargo_config:
```

Comment:
Deleting this allows the rust code used for MODULE.bazel to be the source of truth for both legacy WORKSPACEs and MODULE.bazel